cape-ph/cape-cod
# 190 update default config file docs and values

# TO TEST
Review. There is one code change to force a failure on a missing config key, but otherwise should be nothing but doc changes.

There was a deployment associated with this branch with a single change. The description of `ccd-pvsl-dap-api-lsdaps-lmbdfn` changed due to the description being moved to the config file to provide an example (previously it was defaulted and not in the config at all).
